### DISCUSSION TITLE
Light tracing shadow rendering

### DISCUSSION THREAD

**Discussion by WdsPjm** [vsg-dev/VulkanSceneGraph#1001] [General]
I implemented ray-tracing shadows based on vsgraytrace.cpp and am currently adding pbr materials to them. I want to show you this code in the hope that it will be helpful to others. But I don't know what to do.

![811b02657add2592bc40bf0a50a6378](https://github.com/vsg-dev/VulkanSceneGraph/assets/52599568/cc4fd873-0bd9-4905-9301-6cc29a2e8f5c)

**Comment by robertosfield** [comment]
What part do you need help with? Publishing the code?

It would be great if the work could become the new vsgrayracing example in vsgExamples, cloning the vsgExamples then making changes to it then check your changes into your clone repo, then use github's interface for generating a PR on the main repo.  Github will have docs on all these steps.

**Reply by WdsPjm** [reply]
I also modified the code related to vsg and light tracing, which was not very reasonable. I don't know how to handle that part. But for now it's working.
